.travis.yml

```diff
@@ -1,6 +1,15 @@
 language: node_js
-
 node_js:
-- "8.9.0"
-
+  - "8.9.0"
 sudo: false
+install:
+  - npm install
+deploy:
```

Comment:
When is this script executed? Can I set up a webhook from the original repo to trigger this job? I suppose I can set up a curl from the CI of the main repo https://docs.travis-ci.com/user/triggering-builds/

Reply:
We can trigger a build with travis itself. It should trigger a build on this repo at that time. https://hiddentao.com/archives/2016/08/29/triggering-travis-ci-build-from-another-projects-build/